metadata.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "herculesteam-augeasproviders_sysctl",
+  "name": "puppet-augeasproviders_sysctl",
   "version": "2.6.2",
   "author": "Raphael Pinson, Trevor Vaughan",
   "summary": "Augeas-based sysctl type and provider for Puppet",
   "license": "Apache-2.0",
-  "source": "https://github.com/hercules-team/augeasproviders_sysctl",
-  "project_page": "https://github.com/hercules-team/augeasproviders_sysctl",
-  "issues_url": "https://github.com/hercules-team/augeasproviders_sysctl/issues",
+  "source": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
+  "project_page": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
+  "issues_url": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl/issues",
   "description": "This module provides type/provider for sysctl using the Augeas configuration API library.",
   "dependencies": [
     {
-      "name": "herculesteam/augeasproviders_core",
```

Comment:
Just bear in mind that this means bumping dependency to:
```
"version_requirement": ">= 3.2.0 < 4.0.0"
```
There's no other release of [puppet/augeasproviders_core](https://forge.puppet.com/modules/puppet/augeasproviders_core/readme) for backwards compatibility.

Reply:
```
Module herculesteam-augeasproviders_core found versions: 3.1.0, 3.0.0, 2.7.0, 2.6.0, 2.5.0, 2.4.0, 2.3.0, 2.2.0, 2.1.5, 2.1.4, 2.1.3, 2.1.2, 2.1.1, 2.1.0, 2.0.1, 2.0.0
```
librarian seems to be able to deal with this change:
```
Dependency on module 'augeasproviders_core' is fullfilled by multiple modules and only one will be used: ["herculesteam-augeasproviders_core", "puppet-augeasproviders_core"]
```